src/env.h

```diff
@@ -578,6 +581,9 @@ class Environment {
 
   char* http_parser_buffer_;
 
+  struct AtExitCallback;
+  AtExitCallback* at_exit_functions_ = nullptr;
```

Comment:
For the record, I'd be okay with switching this a `std::list<...>`.  Could be a separate commit or pull request.